.github/workflows/cygwin.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   cygwin_build_test:
     runs-on: windows-latest
     # To enable this workflow on a fork, comment out:
-    if: github.repository == 'numpy/numpy'
+    # if: github.repository == 'numpy/numpy'
```

Comment:
```suggestion
    if: github.repository == 'numpy/numpy'
```
Accept this suggestion before merging

Reply:
I thought I was making this PR on another repository.  This should probably be marked as draft until then